naples/utils.py

```diff
@@ -84,7 +87,7 @@ def createMsgEmailChangePassword(token: str, verify_router: str) -> str:
     return html_content
 
 
-def sendEmail(email: str, message: str, ses_client: SESClient):
+def sendEmail(email: str, message: str, ses_client: SESClient, db: Session):
```

Comment:
So if we fail to send ANY email, we delete the user????

Reply:
why do we need to pass user email as argument, if we can pass user model directly?

Reply:
yes, you are right, I will add the user model